langlib/lang.table/src/main/ballerina/table.bal

```diff
@@ -134,6 +224,15 @@ public isolated function forEach(table<MapType> t, @isolatedParam function(MapTy
 #
 # The resulting table will have the same keys as the argument table.
 #
+# ```ballerina
+# table [
+#     {id: 1, salary: 1200},
+#     {id: 2, salary: 1100},
+#     {id: 3, salary: 800}
+# ]
```

Comment:
Follow the below suggestion in all places

Reply:
https://github.com/ballerina-platform/ballerina-lang/pull/39084#discussion_r1054075181